modules/virt-features-for-storage-matrix.adoc

```diff
@@ -6,32 +6,89 @@
 = {VirtProductName} storage feature matrix
```

Comment:
I can't comment higher up but can you please add these two assemblies in the module metadata

Reply:
Oops. Missed that. Done.